ReportingServicesTools.psm1

```diff
@@ -1,2 +1,17 @@
+﻿<#
+    .NOTES
+    --------------------------------------------------------------------------------
+     Code generated by:  SAPIEN Technologies, Inc., PowerShell Studio 2017 v5.4.135
+     Generated on:       05.02.2017 14:43
+     Generated by:       Friedrich Weinmann
```

Comment:
We don't have licenses for such tool, please check their license agreement and validate it could be used for this open source project